File: Backend/TrackrAPI/TrackrAPI/Dtos/GestionExpediente/PadecimientoDTO.cs
using TrackrAPI.Models;

namespace TrackrAPI.Dtos.GestionExpediente;

    public class PadecimientoDTO
    {
        /*public string? IconoClase {get; set;} */
        public int IdPadecimiento { get; set; }
        public string NombrePadecimiento { get; set; }
        public List<VariablesPadecimientoDTO> Variables { get; set; }
        /*public int? IdWidget { get; set; } */
        public string DescripcionWidget { get; set; }
    }

File: Frontend/trackr-paciente/src/app/shared/Dtos/gestion-expediente/padecimiento-dto.ts
import { VariablesPadecimientoDTO } from "./variables-padecimiento-dto";


export class PadecimientoDTO
{
    idPadecimiento: number;
    nombrePadecimiento: string;
    variables : VariablesPadecimientoDTO[];
    idWidget : number;
    descripcionWidget : string;
    tomasTomadas : number;
    tomasTotales : number;
}